api/test/wfl/unit/utils_test.clj

```diff
@@ -74,3 +74,11 @@
                       [:membership "flowcell_set" membership]
                       [:membership "flowcell_set_id" membership]]]
       (run! go parameters))))
+
+(deftest test-trim-slashes
+  (is (= "wfl" (util/trim-slashes "/wfl")))
+  (is (= "wfl" (util/trim-slashes "/wfl/")))
+  (is (= "wfl" (util/trim-slashes "wfl/")))
+  (is (= "wfl" (util/trim-slashes "wfl")))
+  (is (= "" (util/trim-slashes "/")))
+  (is (= "" (util/trim-slashes "///"))))
```

Comment:
We can get rid of `util/trim-slashes` and its test if you agree with my comments above.